docs/contributor_guide/contributing_example_notebooks.rst

```diff
@@ -13,4 +15,39 @@ include:
 Good examples of existing notebooks that abide by these requirements are:
 
 * `Mitigating Disparities in Ranking from Binary Data <https://github.com/fairlearn/fairlearn/blob/master/notebooks/Mitigating%20Disparities%20in%20Ranking%20from%20Binary%20Data.ipynb>`_
-* `Binary Classification with the UCI Credit-card Default Dataset <https://github.com/fairlearn/fairlearn/blob/master/notebooks/Binary%20Classification%20with%20the%20UCI%20Credit-card%20Default%20Dataset.ipynb>`_
+* `Binary Classification with the UCI Credit-card Default Dataset <https://fairlearn.github.io/auto_examples/notebooks/plot_binary_classification_uci_credit_card_default.html>`_
+
+Working with example notebooks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The example notebooks in Fairlearn are surfaced through the website's
+`example notebook page <https://fairlearn.github.io/auto_examples/notebooks/index.html>`_.
+This allows them to be rendered properly with output from all cells.
+
+.. note:
+
+    Rendering the Fairlearn dashboard is still an outstanding issue.
+
+These notebooks are generated based on `.py` files in the
+`examples/notebooks directory <https://github.com/fairlearn/fairlearn/tree/master/examples/notebooks>`_
+of the repository. To do this yourself make sure to install sphinx and its
+add-ons by running :code:`pip install -r requirements.txt` in the repository
+root directory.
+
+When working on the notebooks it can be tedious to make changes in the
+notebook and then make the same change in the `.py` file. To make the
+development process smoother we recommend using 
+`Visual Studio Code <https://code.visualstudio.com/docs/python/jupyter-support>`_.
+VS Code recognizes the lines starting with :code:`$ %%` as new cells.
+Each cell can be executed individually by clicking on *Run Cell*, and VS Code
+opens a *Python Interactive* tab to show the output.
+
+.. image:: ../_static/images/vscode-jupyter.png
+
+Note that the text portion of these notebooks should be written in
```

Comment:
This paragraph says "use ReST but don't use ReST." I'm guessing that the tool which produces the notebooks from the `.py` files doesn't know how to translate ReST into the variant of markdown used by the notebooks?

Reply:
It does, but only for certain things. I've asked the sphinx-gallery folks about this and they're working on it, but I wouldn't expect this anytime soon. That's why I'd rather have the disclaimer there.

Reply:
It might be more helpful to give a list of the ReST markup which works.

Reply:
That's fair, although I don't think it's worth spending much time on that since sphinx-gallery folks already fixed this problem and we're just waiting for their next release. Once that's out I'll try a few things (links, more complicated directives, etc.) and if any of them don't work yet I'll be happy to update. Does that make sense?

Reply:
I've upgraded to `sphinx-gallery==0.8.1` and activated `pypandoc` but that doesn't really seem to be helping. Perhaps it's restricted to certain types of rest-to-md conversions...